Kitodo/src/main/webapp/pages/userInstructions.xhtml

```diff
@@ -0,0 +1,25 @@
+<ui:composition
```

Comment:
A few comments to this whole new file:
- Missing license header. Please add the default license.
- Please check formatting. At least in `h:output` line is a whitespace to much before `styleClass` attribute.
- Please removing unnecessary empty lines.